starcluster/plugins/ipcluster.py

```diff
@@ -163,7 +165,10 @@ def _write_config(self, master, user, profile_dir):
         f.close()
 
     def _start_cluster(self, master, profile_dir):
-        n_engines = max(1, master.num_processors - 1)
+        if not self.master_engines: 
```

Comment:
Hmm... this works in testing. Let me figure this out. 

Oh, actually, at this point in the execution, this is the string '0', which evaluates to true. This is bad and confusing, though, so I will change to check for "None". Thanks.